crates/gpui/src/platform/windows/window.rs

```diff
@@ -994,6 +995,11 @@ impl WindowsWindowInner {
 
         LRESULT(0)
     }
+
+    fn handle_set_cursor(&self) -> LRESULT {
```

Comment:
> `WM_SETCURSOR`: Sent to a window if the mouse causes the cursor to move within a window and mouse input is not captured.

I dont think it is the event that we call `SetCursor`

Reply:
No, `DefWindowProc` is call `SetCursor` on this event to set default window class related cursor.